analyzers/tests/SonarAnalyzer.UnitTest/TestCases/EmptyNestedBlock.cs

```diff
@@ -81,6 +82,14 @@ void F1(bool b)
         void F2()
         {
         }
+
+        // See: https://github.com/SonarSource/sonar-dotnet/issues/4540
+        public void ConditionalCompilation(object obj)
```

Comment:
If I understood correctly this rule looks only at nested blocks. So, the FP can be reproduced with the following code example:
```
        // See: https://github.com/SonarSource/sonar-dotnet/issues/4540
        public void ConditionalCompilation(object obj)
        {
            if (obj != null)
            {
#if DEBUG
                Trace.WriteLine("message");
#endif
            }
        }
```

Reply:
Thanks. I've took the example from the issue without reading the rule description thoroughly.